content/millennium/r4/individuals/practitioner.md

```diff
@@ -37,14 +37,17 @@ Search for Practitioners that meet supplied query parameters:
 
 ### Parameters
 
- Name        | Required?            | Type      | Description
--------------|----------------------|-----------|-------------------------------------------------------
- `_id`       | This or `identifier` | [`token`] | The logical resource id associated with the resource.
- `identifier`| This or `_id`        | [`token`] | A practitioner identifier/alias. Example: `http://hl7.org/fhir/sid/us-npi|4326587548`
+ Name        | Required?                                     | Type       | Description
+-------------|-----------------------------------------------|------------|---------------------------------------------------------------------------------
+ `_id`       | This or `identifier` or `family`              | [`token`]  | The logical resource id associated with the resource.
+ `identifier`| This or `_id` or `family`                     | [`token`]  | A practitioner identifier/alias. Example: `http://hl7.org/fhir/sid/us-npi|4326587548`
+ `family`    | This and/or `given`, or `_id` or `identifier` | [`string`] | The start of the family name of the practitioner. Example: `Smith`
```

Comment:
I dont think the 'and/or given' makes sense here because it cant be 'or' because given cant be without family. Im not sure the best way to word it though

Reply:
I agree. Right now, I'll update the others to `This, or any other required search parameter`. As for 'given' -> `This and `family`, or any other required search parameter` does this make sense?

Reply:
I think we should match the Patient resource.
This and/or any other search param, or _id

Reply:
@kathypickering We cannot say `and/or` here as we don't allow either of `_id`, `identifier` or `family` set together. And `given` can be set only with `family`.

So right now i think, for `_id`, `identifier` & `family`:
`This or any other required search parameter`
For `given`:
`This and `family`, or any other required search parameter`

Does this look good?

Reply:
+1

Reply:
updated https://github.com/cerner/fhir.cerner.com/pull/538/commits/b451d7a308fc4f58f082228aa64fb652f5f9c2f7